### PR TITLE
Storage max size unit tests

### PR DESCRIPTION
**Pull request by olmisa**
* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you add unit tests if this modifies the Windows code?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Unit tests for set storage max size functionality were added.

## Related PRs or issues

[AB#83807](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/83807)

### CODE REVIEW

**Review by MatkovIvan — APPROVED** [review]

**Review by AnastasiaKubova — APPROVED** [review]

File: Tests/Microsoft.AppCenter.Test.Windows/Storage/StorageAdapterTest.cs [hunk 1]
```diff
@@ -178,6 +182,56 @@ public void CreateTableAndInsertWorking()
...
         }
 
+        /// <summary>
+        /// Verify SetMaxStorageSize sets db size.
+        /// </summary>
+        [TestMethod]
+        public void SetMaxStorageSize()
+        {
+            InitializeStorageAdapter();
+
```
Comment (AnastasiaKubova) [inline Tests/Microsoft.AppCenter.Test.Windows/Storage/StorageAdapterTest.cs:192]: Please remove empty lines or add a comments

Comment (olmisa) [reply Tests/Microsoft.AppCenter.Test.Windows/Storage/StorageAdapterTest.cs]: Fixed

File: Tests/Microsoft.AppCenter.Test.Windows/Utils/StorageTestUtils.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            raw.sqlite3_prepare_v2(db, "PRAGMA page_size;", out stmt);
+            raw.sqlite3_step(stmt);
+            var pageSize = raw.sqlite3_column_int(stmt, 0);
+            raw.sqlite3_finalize(stmt);
+            raw.sqlite3_close(db);
+
+            return (long)pageCount * pageSize;
+        }
+
+
```
Comment (AnastasiaKubova) [inline Tests/Microsoft.AppCenter.Test.Windows/Utils/StorageTestUtils.cs:40]: empty line

File: Tests/Microsoft.AppCenter.Test.Windows/Storage/StorageTest.cs [hunk 3]
```diff
@@ -365,6 +371,122 @@ public async Task DoNotRecreateCorruptedDatabaseOnNotCorruptException()
...
+        /// </summary>
+        [TestMethod]
+        public async Task SetMaxStorageSize()
+        {
+            var mockStorageAdapter = Mock.Of<IStorageAdapter>();
+            using (var storage = new Microsoft.AppCenter.Storage.Storage(mockStorageAdapter, _databasePath))
+            {
+                var dbSize = 2 * 1024 * 1024;
+                _ = await storage.SetMaxStorageSizeAsync(dbSize);
+
```
Comment (AnastasiaKubova) [inline Tests/Microsoft.AppCenter.Test.Windows/Storage/StorageTest.cs:385]: empty lines in test methods

File: Tests/Microsoft.AppCenter.Test.Windows/AppCenterTest.cs [hunk 4]
```diff
@@ -848,6 +848,73 @@ public void SetWrapperSdk()
...
+        /// Verify that AppCenter forwards SetMaxStorageSize to its channelGroup.
+        /// </summary>
+        [TestMethod]
+        public async Task SetMaxStorageSize()
+        {
+            var dbSize = 2 * 1024 * 1024;
+            var storageTask = AppCenter.SetMaxStorageSizeAsync(dbSize);
+            AppCenter.Start("appsecret", typeof(MockAppCenterService));
+            await storageTask;
+
```
Comment (AnastasiaKubova) [inline Tests/Microsoft.AppCenter.Test.Windows/AppCenterTest.cs:862]: empty lines in test methods

**Pull request merged**